framework/src/executioners/Transient.C

```diff
@@ -261,9 +305,72 @@ Transient::execute()
 }
 
 void
-Transient::computeDT()
+Transient::updateStepperInfo(bool first)
+{
+  _soln_nonlin.resize(_fe_problem.getNonlinearSystem().currentSolution()->size());
+  _soln_aux.resize(_fe_problem.getAuxiliarySystem().currentSolution()->size());
+  _soln_predicted.resize(_fe_problem.getNonlinearSystem().currentSolution()->size());
+
+  if (first)
+    _si.pushHistory(_prev_dt, true, 0);
+  _si.update(
+    _steps_taken,
+    _time,
+    _dt,
+    _nl_its,
+    _l_its,
+    _last_solve_converged,
+    _solve_time,
+    _soln_nonlin,
+    _soln_aux,
+    _soln_predicted
+  );
```

Comment:
Is this clang format here? I don't think I like separate lines for each argument.